packages/terra-form-radio/src/RadioField.jsx

```diff
@@ -109,13 +113,17 @@ const RadioField = (props) => {
     legendAttrs.className,
   ]);
 
+  const feildSetId = `terra-radio-group-${uniqueid()}`;
   const legendAriaDescriptionId = `terra-radio-field-description-${uniqueid()}`;
   const helpAriaDescriptionId = help ? `terra-radio-field-description-help-${uniqueid()}` : '';
   const errorAriaDescriptionId = error ? `terra-radio-field-description-error-${uniqueid()}` : '';
   const ariaDescriptionIds = `${legendAriaDescriptionId} ${errorAriaDescriptionId} ${helpAriaDescriptionId}`;
 
+  const isSafariOREdge = ((navigator.userAgent.indexOf('Safari') !== -1 && navigator.userAgent.indexOf('Chrome') === -1) || navigator.userAgent.indexOf('Edg') !== -1);
```

Comment:
Why is the condition for Chrome needed? It is redundant to check for it when both this condition and checking for safari will always return true. If the Safari condition is false, then the chrome check will never be evaluated due to short circuit evaluation.

Reply:
Can we add a comment since it is not easy to understand just from the code why the Chrome check is necessary?

Also, can we split the `isSafariOREdge` variable into two variables for better readability since it can have multiple implications? Then on [line 123](https://github.com/cerner/terra-core/pull/3804/files#diff-de27e9ca32868da5500732f9a44baeb41aa7159e7a4bd287c0149be277869b2dR123), use `isSafari || isEdge` in the ternary condition.

Something like:

```suggestion
  // The Chrome check is necessary to filter out Chrome browsers on MacOS
  const isSafari = navigator.userAgent.indexOf('Safari') !== -1 && navigator.userAgent.indexOf('Chrome') === -1;
  
  const isEdge = navigator.userAgent.indexOf('Edg') !== -1;
```